CONDUCT.adoc

```diff
@@ -0,0 +1,38 @@
+= Code of Conduct
+
+== Introduction
+
+This code of conduct applies to all spaces related to Peco, including chat, mailing lists, issue trackers, wikis, blogs, social networks, and any other communication channels our community use.
+We expect everyone who participates in the Peco community formally or informally, or claims any affiliation with Peco to honor this code of conduct.
+
+This code is not exhaustive or complete. It distills our common understanding of a collaborative, shared environment and goals. We expect all participants to the Peco community to follow it in spirit as in the letter, so that it can enrich all of us.
+
+== Specific guidelines
+
+We strive to:
+
+1. *Be open*. We invite anyone to participate in our community. We prefer to use public methods of communication for project-related messages, unless discussing something sensitive. This applies to messages for help or project-related support, too; not only is a public support request much more likely to result in an answer to a question, it also makes sure the community notices and corrects any inadvertent mistakes people answering the query may make.
+2. *Be empathetic, welcoming, friendly, and patient*. We work together to resolve conflicts, assume good intentions, and do our best to act in an empathetic fashion. We may all experience some frustration from time to time, but we do not allow frustration to result in a personal attack. A community where people feel uncomfortable or threatened is not a productive one. We should be respectful when dealing with other community members as well as with people outside our community.
+3. *Be collaborative*. Other people will use our work, and we in turn depend on the work of others. When we make something for the benefit of the project, we are willing to explain to others how it works, so they can build on the work to make it even better. Any decision we make will affect users and colleagues, and we take those consequences seriously when making decisions.
+4. *Be inquisitive*. Nobody knows everything! Asking questions early avoids many problems later, so we encourage questions, although we may redirect them to the appropriate forum. Those who receive a question should be responsive and helpful, within the context of our shared goal of improving Apache project code.
+5. *Be careful in the words that we choose*. Whether we are participating as professionals or volunteers, we value professionalism in all interactions, and take responsibility for our own speech. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behaviour are not acceptable. This includes, but is not limited to:
+
+----
```

Comment:
```suggestion
```

Reply:
Enclosing the bullet list in `----` makes it a code block, and I don't think it is the format you want